src/lib/ibcclient.ts

```diff
@@ -257,6 +272,34 @@ export class IbcClient {
     this.logger = logger ?? new NoopLogger();
   }
 
+  public revisionHeight(height: number): Height {
+    return Height.fromPartial({
+      revisionHeight: new Long(height),
+      revisionNumber: this.revisionNumber,
+    });
+  }
+
+  public ensureRevisionHeight(height: number | Height): Height {
+    let revisionHeight: Long;
+    const revisionNumber = this.revisionNumber;
+    if (typeof height === 'number') {
+      revisionHeight = Long.fromNumber(height);
+    } else {
+      if (height.revisionNumber.toNumber() !== revisionNumber.toNumber()) {
+        throw new Error(
+          `Using incorrect revisionNumber ${height.revisionNumber} on chain with ${this.revisionNumber}`
+        );
+      }
+      revisionHeight = height.revisionHeight;
+    }
+    return Height.fromPartial({ revisionHeight, revisionNumber });
+  }
```

Comment:
This seems simpler?
```ts
if (typeof height === 'number') {
  return Height.fromPartial({ revisionHeight: new Long(height), revisionNumber: this.revisionNumber });
}
if (height.revisionNumber.toNumber() !== this.revisionNumber.toNumber()) {
  throw new Error("xxx");
}
return height;
```

Reply:
Yes, simpler.

Note: If you are not going to be at the computer and responsive and it is a small, clear change, can you just approve it? I will merge anyway (admin rights), but we have a pile up of PRs and I think a bit of pre-emptive approvals (when you can trust the dev to just implement the minor change requests) is a better flow.